packages/datadog-plugin-fs/test/index.spec.js

```diff
@@ -622,7 +622,7 @@ describe('Plugin', () => {
 
         it('should handle errors', () => {
           testHandleErrors(fs, 'ReadStream', (fs, args, _, cb) => {
-            fs.createReadStream(...args).on('error', cb).emit('error', new Error('bad'))
+            fs.createReadStream(...args).on('error', cb)
```

Comment:
The error was being emitted to ensure errors are being handled correctly. Removing it makes this test useless.

Maybe we can find a more deterministic way of mimicking the error?

Reply:
I assumed there were 2 errors since the code already looked deterministic to me. The weird thing is that the error is not the one that is manually emitted, but an actual error while trying to load the file, which I assumed was the second error.

Do you know why the test would pass when the error is not thrown?

Reply:
I added back the error and updated the `afterEach` to swallow errors since we're causing a race condition in the error handling test.